falcon/response.py

```diff
@@ -247,8 +247,16 @@ def set_cookie(self, name, value, expires=None, max_age=None,
     def unset_cookie(self, name):
         """Unset a cookie from the response
```

Comment:
Would you mind expanding the docstring with a note talking about how this method will also instruct the user agent to expire it's copy of the cookie, if previously set?